packages/create-cloudflare/e2e-tests/pages.test.ts

```diff
@@ -137,10 +137,10 @@ describe.concurrent(`E2E: Web frameworks`, () => {
 
 	// These are ordered based on speed and reliability for ease of debugging
 	const frameworkTests: Record<string, FrameworkTestConfig> = {
-		astro: {
-			expectResponseToContain: "Hello, Astronaut!",
-			testCommitMessage: true,
-		},
+		// astro: {
+		// 	expectResponseToContain: "Hello, Astronaut!",
+		// 	testCommitMessage: true,
+		// },
```

Comment:
😱 ??

Does this mean this test is now failing?

Reply:
But how come it only needs to be commented out in this PR? Should this be failing on `main` already?

This brings up a point we have been discussing on C3. Should we bump the min node.js version to 18? That is a breaking change so would require a major version bump of C3. I think ideally C3 and Wrangler should bump min node.js version at the same time. Are there plans for that in Wrangler?

Reply:
It's not failing on `main` since this test only runs on PRs that touch code under `packages/create-cloudflare/`, but it is failing on other PRs right now, I think. Maybe these tests should run on `main` as well?

Re a major version bump—I think so. We're still discussing it, but I think we're aiming for a 6-monthly cadence (so mid-November for v4),  and I think the plan is to drop Node 16 support for Wrangler as part of that (since it's EOL next week)